src/scss/generics/inputgroup.scss

```diff
@@ -0,0 +1,7 @@
+.inputgroup {
```

Comment:
Could you put these styles into that file instead (and remove this file):
https://github.com/HerrZatacke/gb-printer-web/blob/master/src/javascript/app/components/Input/index.scss

Reply:
Oops, I only looked under scss so I missed that file.